src/OpenTK.Mathematics/OpenTK.Mathematics.csproj

```diff
@@ -3,6 +3,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Deterministic>true</Deterministic>
+    <RootNamespace>OpenToolkit.Mathematics</RootNamespace>
```

Comment:
Can we call this Math? It's a little less typing...

Reply:
Changed the namespace